react/components/FilterBar/Atoms/SubjectAtom.js

```diff
@@ -54,14 +54,23 @@ class SubjectAtom extends React.Component {
 
     return (
       <div className={`mh3 ${isFullWidth ? 'pb3' : ''}`}>
-        <Dropdown
+        <Select
           ref={forwardedRef}
           placeholder={placeholder}
           options={uniqueOptions}
-          value={!condition.subject ? '' : condition.subject || ''}
-          onChange={(e, value) => {
-            this.handleChangeStatement(value, 'subject')
+          defaultValue={
```

Comment:
I don't usually use defaultValue, I use the value prop. Generally the select value is passed as a prop and should be used as the prop value in my opinion, but we can discuss your use case.